project/cmake/CMakeLists.txt

```diff
@@ -199,14 +205,22 @@ endif()
 if(NOT WIN32)
   core_optional_dep(OpenGl)
   if(OPENGL_FOUND)
-    core_optional_dep(X ENABLE_X11)
-    core_optional_dep(LibDRM ENABLE_X11)
-    core_optional_dep(XRandR ENABLE_X11)
+    if (ENABLE_MIR)
```

Comment:
Good reminder that I need to clean up this spaghetti code. Ok for now, but make it `if(` without the space everywhere please.